package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@google-cloud/cloud-sql-connector",
-      "version": "0.2.0",
```

Comment:
does this PR need to be rebased or something? This change is already in main??

Reply:
not really! this is info internal to `package-lock.json` and it looks like it only gets updated after a `npm install` following the version bump, I should go back to the npm cli and fix that one day but not today 😄